aredis_om/model/model.py

```diff
@@ -32,7 +32,7 @@
 from ulid import ULID
 
 from .. import redis
-from .._compat import BaseModel
+from .._compat import PYDANTIC_V2, BaseModel, ConfigDict
```

Comment:
need to move this import statement, it will break if pydantic v2 isn't present since it's conditionally exporting it in _compat

Reply:
@slorello89 Updated. Please have a look again. Thanks!